From 56babea2cfc6a324f89d938f21a0b6c5e9d633d8 Mon Sep 17 00:00:00 2001
From: Orion Poplawski <orion@cora.nwra.com>
Date: Tue, 17 Mar 2015 16:02:00 -0600
Subject: [PATCH] Create $(DESTDIR)$(BINDIR) if needed (i.e. when building rpm
 package
---
diff --git a/scripts/Makefile.in b/scripts/Makefile.in
index 5519a2b..806227f 100644
--- a/scripts/Makefile.in
+++ b/scripts/Makefile.in
@@ -26,4 +26,5 @@ distclean: clean
 	rm -f Makefile
 
 install:
+	$(INSTALL) -m 0755 -d $(DESTDIR)$(BINDIR)
 	$(INSTALL) -m 0755 mod_python $(DESTDIR)$(BINDIR)/mod_python
-- 
2.28.0

